app/main.py

```diff
@@ -1,3 +1,70 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> float:
+        self.km = km
+
+    def __str__(self) -> float:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> float:
```

Comment:
The `__repr__` method should have a return type annotation of `str`, not `float`, as it returns a string representation of the object.